service/metrics.go

```diff
@@ -1,16 +1,42 @@
 package service
 
 import (
+	"net/http"
 	"runtime/debug"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 // MustRegisterMetrics will register all metrics on the given registry.
 func MustRegisterMetrics(registry *prometheus.Registry) {
 	registry.MustRegister(buildInfo)
 }
 
+// MustRegisterHTTPMetrics will register all HTTP related metrics on the given registry.
+func MustRegisterHTTPMetrics(registry *prometheus.Registry) {
```

Comment:
Separated since a few services may not expose HTTP or expose other protocols in the future.